Examples/Modules/boosted_diags/inputs_3d

```diff
@@ -104,14 +102,6 @@ laser1.profile_t_peak = 40.e-15    # The time at which the laser reaches its pea
 laser1.profile_focal_distance = 0.5e-3  # Focal distance from the antenna (in meters)
 laser1.wavelength = 0.81e-6         # The wavelength of the laser (in meters)
 
-slice.dom_lo     = xmin  0.0  zmin
-slice.dom_hi     = xmax  0.0  zmax
-slice.coarsening_ratio = 1 1 1
-slice.plot_int = -1
-slice.num_slice_snapshots_lab = 4
-slice.dt_slice_snapshots_lab = 3.3356409519815207e-12
-slice.particle_slice_width_lab = 2.e-6
-
```

Comment:
Slicing temporarily removed from the test, since it's not yet supported by the new BTD, see #3512. Test, input file, and analysis script renamed for consistency. Once slicing is added back in the new BTD, we can add a specific test for it, or include it in one of the existing tests.